src/lib/api/services/upload.service.ts

```diff
@@ -26,7 +26,7 @@ export default {
             throw error;
         }
     },
-    uploadFile: async (filepath, namespace, file_name = null) => {
+    uploadFile: async (filepath, namespace, file_name = null, mimeType = null) => {
```

Comment:
@karanrainafynd @ashokrdkr In new implementation of grindor, No name file, File name without extension and Files with extension mismatch as explicitly passed here won't work. We need to update our implementation here.

FYI @ZaidRehmanFynd

Reply:
@Shivraj0 Updated PR.

We will now have 2 files - ( *.jpeg and *.js ) for image and asset upload URLs.